app/components/super-features/super-features-relations/super-features-relations.html

```diff
@@ -1,7 +1,7 @@
 <div class="super-features-relations-list-header">
   <h4>Child Pages</h4>
 
-  <div ng-if="relations.length > 0">
+  <div ng-if="article.child_types.length > 0">
```

Comment:
Does this change need to be reflected in other locations where were accessing `relations` from the scope, and in the directive where we're accessing `$scope.relations`?

If not, I think this will hide tools that we may need even if `article.child_types` is empty.